<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>597</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>652</width>
    <height>597</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>652</width>
    <height>597</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BinaryPlant ARP Defender</string>
  </property>
  <property name="statusTip">
   <string>Hello</string>
  </property>
  <property name="styleSheet">
   <string notr="true">alternate-background-color: rgb(255, 0, 4);
border-color: rgb(255, 152, 26);</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="iconSize">
   <size>
    <width>64</width>
    <height>64</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>550</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget_History">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>300</width>
      <height>371</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">IP (--&gt; MAC list)</string>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="statusTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current ARP table (filtered):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ClearHistory">
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>76</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Clear old records</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>res/clear.ico</normaloff>res/clear.ico</iconset>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Update">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>77</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>Refresh ARP table</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>res/refresh.ico</normaloff>res/refresh.ico</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_gw">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>51</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(175, 39, 29);</string>
    </property>
    <property name="title">
     <string>Default Gateway:</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_gwIp">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0.0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MAC:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_gwMac">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>00-00-00-00-00-00</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>History (this session):</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_Table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>300</width>
      <height>371</height>
     </rect>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string notr="true">Type</string>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MAC</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_Protect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-color: rgb(255, 75, 192);
gridline-color: rgb(255, 82, 151);
alternate-background-color: rgb(71, 35, 255);</string>
    </property>
    <property name="text">
     <string>Can't get gateway IP-MAC pair</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>res/alert.png</normaloff>res/alert.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QCheckBox" name="checkBox_AutoProtect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>76</y>
       <width>301</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Autoprotect (on every timer event)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_StartMinimized">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start minimised to System Tray</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_AutoRun">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Run when Windows starts</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Timer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>356</x>
       <y>49</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>344</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh every:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>362</x>
       <y>77</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>seconds</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>490</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop and Exit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
